rpc/models.py

```diff
@@ -313,19 +312,77 @@ def __str__(self):
 
 
 class FinalApproval(models.Model):
+    """Captures approvals for publication
+
+    This model captures who is asked for approval, when they were asked, and
+    when they gave that approval. Lack of an approved date means approval
+    has not been provided.
+
+    The RPC may need to ask a body, like the IESG, for approval without knowing
+    which person will provide that approval. Body would be non-blank, approver
+    would be None, approved would be None. The body will have a person that
+    approves on their behalf, so once approved is not None, approver will also
+    be not None. Overriding approver will be None when body is used.
+
+    Overriding approver is used when someone has to step in for whoever is
+    pointed to as approver (such as when an author passes away or otherwise
+    becomes non-responsive). Overriding approver should never be not None if
+    approver is None.
+
+    Some approvals are Auth48 approvals from authors. Any FinalApproval with
+    an approval pointing to the same DatatrackerPerson as a RfcAuthor for that
+    RfcToBe is an Auth48 approval.
```

Comment:
I think "and an empty body" is needed to declare something an Auth48 approval to cover the (unlikely and perhaps procedurally dubious but not technically impossible) case that someone who is an author also approves a body approval

Reply:
the key there is procedurally dubious - it won't happen because leadership will watch for that kind of conflict and not let it happen. I'm willing to add "and and empty body" to the docstring, but I dont think it's worth trying to code that into the constraints.

Reply:
(capturing comment from DM for visibility/posterity) That's all I expected. If we later add a manager to provide `FinalApproval.objects.auth48()`, it should perhaps filter out ones with a body.